db/migrate/20160222091312_remove_other_accreditation.rb

```diff
@@ -0,0 +1,12 @@
+class RemoveOtherAccreditation < ActiveRecord::Migration
+  class Accreditation < ActiveRecord::Base
+  end
+
+  def up
+    Accreditation.where(name: 'Other').destroy_all
+  end
+
+  def down
+    fail ActiveRecord::IrreversibleMigration
```

Comment:
Is it really not possible to rollback to a state prior to this migration? Surely it just has no down state?